server.js

```diff
@@ -1,32 +1,146 @@
 import express from "express";
 import cors from "cors";
 import mongoose from "mongoose";
+import netflixData from "./data/netflix-titles.json";
+import dotenv from 'dotenv';
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// import avocadoSalesData from "./data/avocado-sales.json";
-// import booksData from "./data/books.json";
-// import goldenGlobesData from "./data/golden-globes.json";
-// import netflixData from "./data/netflix-titles.json";
-// import topMusicData from "./data/top-music.json";
+dotenv.config();
 
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo";
-mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true });
+
+// from starter code: const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo";
+const mongoUrl = process.env.MONGO_URL || "mongodb://127.0.0.1:27017/project-mongo";
```

Comment:
I guess you've read the thread on Slack, discussing why this change is needed? 
This is also discussed here:
https://github.com/Automattic/mongoose/issues/12366